core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts

```diff
@@ -147,10 +138,10 @@ export class EmaContentletToolsComponent implements OnInit, OnChanges {
      */
     getPosition(): Record<string, string> {
```

Comment:
All these methods that generate styles, used in the template, will be triggered by any movement of the mouse and click. Do you want that behavior?

Reply:
Maybe not, but at the moment it can be left as it is, this ticket is just for drag and drop

Reply:
Those should be pipes tho

Reply:
yes. Good idea, pure pipes.